File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/domain/converter/MlToOzConverter.java
package com.tarbadev.witchcraft.domain.converter;

public class MlToOzConverter implements IConverter {
  private static final Double ML_IN_ONE_OZ = 29.5735;

  @Override
  public Double convert(Double number) {
    return number / ML_IN_ONE_OZ;
  }
}

File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/converter/MlToOzConverter.kt
package com.tarbadev.witchcraft.converter

class MlToOzConverter : IConverter {

    override fun convert(number: Double): Double {
        return number / ML_IN_ONE_OZ
    }

    companion object {
        private const val ML_IN_ONE_OZ = 29.5735
    }
}
